src/sentry/static/sentry/app/components/actions/ignore.tsx

```diff
@@ -11,7 +12,42 @@ import CustomIgnoreDurationModal from 'app/components/customIgnoreDurationModal'
 import ActionLink from 'app/components/actions/actionLink';
 import Tooltip from 'app/components/tooltip';
 
-export default class IgnoreActions extends React.Component {
+enum ModalStates {
+  COUNT,
+  DURATION,
+  USERS,
+}
+
+const IGNORE_DURATIONS = [30, 120, 360, 60 * 24, 60 * 24 * 7];
+const IGNORE_COUNTS = [1, 10, 100, 1000, 10000, 100000];
+const IGNORE_WINDOWS: [number, string][] = [
+  [60, 'per hour'],
+  [24 * 60, 'per day'],
```

Comment:
Should these be translated?

Reply:
They should! There are a bunch of untranslated strings in this component that I want to circle back on.